zetaclient/chains/bitcoin/observer/observer.go

```diff
@@ -341,44 +342,42 @@ func (ob *Observer) WatchGasPrice() {
 // PostGasPrice posts gas price to zetacore
 // TODO(revamp): move to gas price file
 func (ob *Observer) PostGasPrice() error {
-	// hardcode gas price here since this RPC is not available on regtest
-	if chains.IsBitcoinRegnet(ob.Chain().ChainId) {
-		blockNumber, err := ob.btcClient.GetBlockCount()
+	var err error
+	feeRateEstimated := uint64(0)
+
+	// special handle regnet and testnet gas rate
```

Comment:
What does special mean in this context? Should add more details in comment

Reply:
https://github.com/zeta-chain/node/pull/2452/commits/570fe7efb98b8897599e6d8d83844b1be01d7a94